styleguide/source/_patterns/01-molecules/08-lists/12-list-topic-tools/_list-topic-tools.scss

```diff
@@ -4,13 +4,17 @@
   margin: 10px 0;
   padding: 0;
 
+
   // Each list item.
   .list_item-topic-tools {
```

Comment:
Please remove left padding on the LIs at the mobile breakpoint so that the text is flush left with the column edge